corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -53,6 +53,12 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
                     payload[model[index].get('id')] = this.value;
                 }
             });
+            if (hqImport('hqwebapp/js/toggles').toggleEnabled('APP_ANALYTICS')) {
+                hqImport('analytix/js/kissmetrix').track.event('Case list', {
```

Comment:
"Case list" is pretty similar to the other "Viewed case list" event. How about calling this "Case search"?